app/importers/activity_insight_importer.rb

```diff
@@ -883,3 +910,21 @@ def for_external_person?
       activity_insight_user_id.blank?
     end
 end
+
+class ActivityInsightPublicationPostprint
+  def initialize(parsed_postprint)
+    @parsed_postprint = parsed_postprint
+  end
+
+  def postprint_status
+    text_for('ACCESIBLE')
```

Comment:
@nicktiberi Not to be that annoying over-the-shoulder person, but shouldn't it be 'ACCESSIBLE' ?

Reply:
Yes it should be. 🙂 It's just misspelled as "ACCESIBLE" coming from the Activity Insight end, so that's what we have to look for in the XML.

Reply:
I'm literally crying right now.

Reply:
``` ruby
class Accessible
  def to_s
    'ACCESIBLE'
  end
end
```

Reply:
![200](https://user-images.githubusercontent.com/639920/143944743-644a4e63-bf4a-4838-8f65-cd56676bae55.gif)

You're not alone 😁 Ideally this would be fixed on the AI side, but it is what it is for now, I guess.

Reply:
Sorry, I'm crushed by the weight of the irony.